src/h3lib/include/polygonAlgos.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2018 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file polygon.c
```

Comment:
comment doesn't match file name.

I think the comment should explain how this file is intended to be used, especially since it is unlike other headers.

Reply:
Will update.

Reply:
As I believe I've mentioned way back, Doxygen will automatically deduce the file name if you do not provide it. Setting it is equivalent to overriding it for special circumstances (which this is not).

Reply:
From Doxygen docs:

>  If the file name is omitted (i.e. the line after \file is left blank) then the documentation block that contains the \file command will belong to the file it is located in.

https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile